libs/expo/betterangels/src/lib/screens/Note/NoteTitle.tsx

```diff
@@ -13,9 +14,15 @@ export default function NoteTitle({
       <TextBold size="lg" mb="xs">
         {note?.purpose}
       </TextBold>
-      <TextRegular size="sm">
+      <TextRegular size="sm" mb="sm">
         {format(new Date(note?.interactedAt), 'MM/dd/yyyy')}
       </TextRegular>
+      {note?.team && (
```

Comment:
@mikefeldberg  - just a small FYI - in these conditional renders it's best to always check a boolean, so here I'd do - `!!note?.team`. Not a drastic thing, but a 0 would actually render the number 0 here.

Reply:
good call 🙏